### PR TITLE
initial add-on detail page

### PR DESCRIPTION
**Pull request by ngokevin**
screen shot 2015-08-27 at 2 06 09 pm
screen shot 2015-08-27 at 2 06 03 pm

### CODE REVIEW

File: src/media/js/routes.js [hunk 1]
```diff
@@ -4,6 +4,8 @@ define('routes',
 
     router.addRoutes([
         {'pattern': '^/(app.html|index.html)?$', 'view_name': 'homepage'},
+
```
Comment (chuckharmston) [inline src/media/js/routes.js]: Extra newline.

File: src/media/js/views/addon.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+define('views/addon',
```
Comment (chuckharmston) [inline src/media/js/views/addon.js]: We should probably 404 this if `addonsEnabled` is false.

**Comment by chuckharmston** [comment]
community_image_1382806384

Looks good. Very skeleton-y. r+wc

**Comment by chuckharmston** [comment]
r- your Persona.

File: src/media/js/views/addon.js (hunk 2 above)
Comment (ngokevin) [inline src/media/js/views/addon.js]: don't want to do extra work for unnecessarily hiding everything

**Comment by ngokevin** [comment]
put your r- up your skeleton

**Pull request merged**